src/include/stir/IO/InputStreamFromROOTFileForECATPET.inl

```diff
@@ -21,29 +21,29 @@ int
 InputStreamFromROOTFileForECATPET::
 get_num_rings() const
 {
-    return this->crystal_repeater_z;
+   return this->block_repeater_y * this->crystal_repeater_z;
```

Comment:
`block_repeater_z`?

Reply:
Yes, i was about to fix it

Reply:
I think there is an example in https://github.com/UCL/STIR/blob/9be73d4615e4cd53a6fbc2abdd6ebf4e8f85e59d/src/include/stir/IO/InputStreamFromROOTFileForECATPET.h#L51-L66

i added the keyword and also corrected the current crystal keywords that were wrong.